README.md

```diff
@@ -1,5 +1,5 @@
 > **Warning**
-> 
```

Comment:
Prettier did a bunch of auto-formatting on some of this locally, sorry about that. 😬